### PR TITLE
5th and last post of the birder series

### PR DESCRIPTION
**Pull request by maelle**
https://deploy-preview-254--ropensci.netlify.com/blog/2018/09/11/birds-science/

### CODE REVIEW

**Comment by maelle** [comment]
Again, maybe a too simplistic post 🤔

File: content/blog/2018-09-11-birds-science.md [hunk 1]
```diff
@@ -0,0 +1,454 @@
...
+scientific literature &data and data publication,
+[here](https://ropensci.org/packages/).
+
+#### No more birding? No, your turn!
+
+This was the last post of this series, that hopefully provided an
+overview of how rOpenSci packages can help you learn more about birds,
+and can support your workflow. As a reminder, in this series we saw
+
+-   [How to identify spot for birding using open geographical
```
Comment (stefaniebutland) [inline content/blog/2018-09-11-birds-science.md]: I think I might love to see a list of packages used in each post, under each post title

File: content/blog/2018-09-11-birds-science.md [hunk 2]
```diff
@@ -0,0 +1,454 @@
...
+-   [How to extract text from old natural history
+    drawings](https://ropensci.org/blog/2018/08/28/birds-ocr/).
+
+-   [How to complement an occurrence dataset with taxonomy and trait
+    information](https://ropensci.org/blog/2018/09/04/birds-taxo-traits/).
+
+-   How to query the scientific literature and scientific open data
+    repositories.
+
+That’s a wrap! But now, don’t *you* hesitate to explore our packages
```
Comment (stefaniebutland) [inline content/blog/2018-09-11-birds-science.md]: you could still finish with "happy birding" :-)

File: content/blog/2018-09-11-birds-science.md (hunk 1 above)
Comment (stefaniebutland) [inline content/blog/2018-09-11-birds-science.md]: "How to identify spot for birding using open geographical data."

should it be "**a** spot..."

File: content/blog/2018-09-11-birds-science.md [hunk 3]
```diff
@@ -0,0 +1,454 @@
...
+Jay, Yellow-legged Gull, Yellowhammer, Eurasian Green Woodpecker, Eared
+Grebe, Eurasian Reed Warbler, Barn Swallow, Northern Shoveler, Eurasian
+Moorhen, Black Redstart, Great Egret, White Stork, Eurasian Wren,
+Long-tailed Tit, Common House-Martin, Eurasian Kestrel, European
+Goldfinch and European Greenfinch
+[(`glue::glue_collapse(species, sep = ", ", last = " and ")`)](https://twitter.com/LucyStats/status/1031938964796657665?s=19).
+
+### Querying the scientific literature
+
+Just like rOpenSci has a taxonomic toolbelt
```
[the remark this answers is not in the document]
Comment (maelle) [reply content/blog/2018-09-11-birds-science.md]: am going to do that! I had lazily copied the description from the repos :joy:

File: content/blog/2018-09-11-birds-science.md [hunk 4]
```diff
@@ -0,0 +1,454 @@
...
+If we were working on a scientific study, we’d add a few more filters,
+e.g. having the species mentioned in the abstract, and not only
+somewhere in the paper which is probably the way the different
+literature search providers define a match. But we’re not, so we can
+keep our query quite free! My favourite paper involving the Carrion Crow
+is [“Investigating the impact of media on demand for wildlife: A case
+study of Harry Potter and the UK trade in
+owls”](http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0182368)
+because it’s a fun and important scientific question, and is supported
+by open data (by the way you can access CITES trade data in R using
```
[the remark this answers is not in the document]
Comment (maelle) [reply content/blog/2018-09-11-birds-science.md]: :+1:

File: content/blog/2018-09-11-birds-science.md [hunk 5]
```diff
@@ -0,0 +1,454 @@
...
+needs to know it’s the silhouette of a bird to recognize it, and we’d
+need to work a bit on not reshaping the silhouette, but it’s fun as it
+is.
+
+### Querying scientific open data
+
+There are quite a few scientific open data repositories out there, among
+which the giant [DataONE](https://www.dataone.org/) that has an API
+interfaced with an R package that’s not part of rOpenSci suite but that
+we can recommend anyway, `dataone`. We shall use it to perform a similar
```
[the remark this answers is not in the document]
Comment (maelle) [reply content/blog/2018-09-11-birds-science.md]: 👍

File: content/blog/2018-09-11-birds-science.md [hunk 6]
```diff
@@ -0,0 +1,454 @@
...
+search that in the next section, but looking at the data indexed on
```
[the remark this answers is not in the document]
Comment (maelle) [reply content/blog/2018-09-11-birds-science.md]: oops I meant "search similar to the previous section", aaall wrong

File: content/blog/2018-09-11-birds-science.md [hunk 7]
```diff
@@ -0,0 +1,454 @@
...
+  
+  tibble::tibble(species = species,
+                 title = result$title,
+                 abstract = result$abstract)
+  }
+}
+```
+
+Note that DataONE searching could be more precise: one can choose to
+search from a data source only for instance. See the [searching DataONE
```
[the remark this answers is not in the document]
Comment (maelle) [reply content/blog/2018-09-11-birds-science.md]: changed to "given data source only"

File: content/blog/2018-09-11-birds-science.md [hunk 8]
```diff
@@ -0,0 +1,454 @@
...
+    vignette](https://github.com/DataONEorg/rdataone/blob/master/vignettes/download-data.Rmd).
+
+### Conclusion
+
+In this post, we used the rOpenSci `fulltext` package, and the DataONE
+`dataone` package, to search for bird species names in scientific papers
+and scientific open datasets. We were able to draw wordclouds
+representing the diversity of topics of studies in which the birds had
+been mentioned or studied. Such a search could be fun to do for your
+favourite bird(s)! And in general, you could have more serious use cases
```
[the remark this answers is not in the document]
Comment (maelle) [reply content/blog/2018-09-11-birds-science.md]: 👍

File: content/blog/2018-09-11-birds-science.md [hunk 9]
```diff
@@ -0,0 +1,454 @@
...
+data include [`europepmc`](https://github.com/ropensci/europepmc), R
+Interface to Europe PMC RESTful Web Service;
+[`jstor`](https://github.com/ropensci/jstor);
+[`suppdata`](https://github.com/ropensci/suppdata) for extracting
+supplemental information, and [much
+more](https://ropensci.org/packages/).
+
+#### Scientific data access… and publication with R
+
+In this post we used the [`dataone`
```
[the remark this answers is not in the document]
Comment (maelle) [reply content/blog/2018-09-11-birds-science.md]: Ok, noted in the tweet drafts issue comment

File: content/blog/2018-09-11-birds-science.md [hunk 10]
```diff
@@ -0,0 +1,454 @@
...
+package](https://github.com/DataONEorg/rdataone) to access data from
+DataONE. That same package allow uploading data to DataONE. The rOpenSci
```
[the remark this answers is not in the document]
Comment (maelle) [reply content/blog/2018-09-11-birds-science.md]: 👍

File: content/blog/2018-09-11-birds-science.md (hunk 1 above)
Comment (maelle) [reply content/blog/2018-09-11-birds-science.md]: Very good point!

**Pull request merged**